packages/salesforcedx-vscode-core/test/vscode-integration/util/authInfo.test.ts

```diff
@@ -79,4 +80,24 @@ describe('OrgAuthInfo', () => {
       infoMessageStub.restore();
     });
   });
+
+  describe('getConnection', () => {
+    const username = 'user@test.test';
+    const alias = 'TestOrg';
+
+    it('with argument', async () => {
```

Comment:
test case conventions include "should" e.g.

`it('should return connection associated with given username')`

`it('should return connection with the default username if one is not provided')`

Reply:
OK. I'm changing them on my next PR. Thanks!